otter/test/convergence/test_gathering.py

```diff
@@ -419,145 +424,128 @@ def test_invalid_first_entry(self):
         self.assertRaises(ValueError, extract_CLB_drained_at, feed)
 
 
+def lb_req(method, url, json_response, response):
```

Comment:
hmm. or not, since append_segments('loadbalancers', '') returns 'loadbalancers/' which breaks the tests. oh well.